final_prep/README.md

```diff
@@ -45,30 +45,39 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
```

Comment:
You will definitely need to block off more time to complete projects outside of the 9-4 timeline every day. I want you to be prepared for that!